### PR TITLE
[iOS] Configure maestro app language

### PR DESCRIPTION
**Pull request by artem888**
## Proposed Changes

Adding additional `language` parameter for LaunchApp command, this way users can test different locales

## Testing
- [x] UIKit app
- [ ] ReactNative app
- [x] Flutter app

### CODE REVIEW

File: maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt [hunk 1]
```diff
@@ -361,7 +361,7 @@ class IntegrationTest {
         driver.assertEvents(
             listOf(
                 Event.StopApp("com.example.app"),
-                Event.LaunchApp("com.example.app")
+                Event.LaunchApp("com.example.app", null)
```
Comment (berikv) [inline maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt:364]: Naming the arguments will help understanding what null means here.

**Review by berikv — APPROVED** [review]

File: maestro-ios/src/main/java/ios/simctl/Simctl.kt [hunk 2]
```diff
@@ -181,15 +189,26 @@ object Simctl {
...
+        var command = mutableListOf(
+            "xcrun",
+            "simctl",
+            "launch",
+            deviceId,
+            bundleId,
+        )
+
+        language?.let {
+            if (supportedLanguages.contains(it)) {
```
Comment (amanjeetsingh150) [inline maestro-ios/src/main/java/ios/simctl/Simctl.kt:202]: Are we throwing exceptions if someone makes an invalid selection?

File: maestro-ios/src/main/java/ios/simctl/Simctl.kt [hunk 3]
```diff
@@ -10,6 +10,14 @@ object Simctl {
 
     data class SimctlError(override val message: String): Throwable(message)
     private val homedir = System.getProperty("user.home")
+    private val supportedLanguages = listOf(
```
Comment (dmitry-zaitsev) [inline maestro-ios/src/main/java/ios/simctl/Simctl.kt:13]: This list seems rather short. Are those really _all_ locales that we support?

File: maestro-ios/src/main/java/ios/simctl/Simctl.kt (hunk 2 above)
Comment (dmitry-zaitsev) [reply maestro-ios/src/main/java/ios/simctl/Simctl.kt:202]: We should certainly inform user that their language selection was invalid

File: maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt [hunk 4]
```diff
@@ -2496,6 +2496,30 @@ class IntegrationTest {
         )
     }
 
+    @Test
+    fun `Case 091 - Launch app with language set`() {
```
Comment (dmitry-zaitsev) [inline maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt:2500]: Please add a test for invalid language codes as well

**Pull request closed without merging**